.claude/CLAUDE.md

```
@@ -31,3 +31,13 @@ The project is an **Nx monorepo**.
 
 - Use early returns whenever possible to reduce nesting and improve readability.
 - Use descriptive variable and function/const names.
+
+## Branch Naming
+
+When creating a branch without a Linear issue, it must match this pattern:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)